tips/totw/55.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Tip of the Week #55: Name Counting and unique_ptr
+layout: tips
+sidenav: side-nav-tips.html
+type: markdown
```

Comment:
Add:

published: true
permalink: tips/55

The above generates it from the _posts/filename -> tips/55